application/src/main/java/org/opentripplanner/routing/api/request/preference/TimeSlopeSafetyTriangle.java

```diff
@@ -25,34 +24,6 @@ public record TimeSlopeSafetyTriangle(double time, double slope, double safety)
 
   public static final TimeSlopeSafetyTriangle DEFAULT = new TimeSlopeSafetyTriangle(1, 1, 1);
 
-  /**
```

Comment:
The JavaDoc on this class is no longer correct - it accepts any 0 or positive values. The values are nomilized in the domain layer.